### PR TITLE
Allow saving of regions and subscriptions

### PR DESCRIPTION
**Pull request by Ladas**

### CODE REVIEW

**Comment by Ladas** [comment]
@agrare ping

File: db/migrate/20181108113645_add_tenant_id_to_regions_and_subscriptions.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddTenantIdToRegionsAndSubscriptions < ActiveRecord::Migration[5.1]
+  def change
+    add_reference :source_regions, :tenant, :type => :bigint, :index => true, :foreign_key => {:on_delete => :cascade}
+    add_reference :subscriptions, :tenant, :type => :bigint, :index => true, :foreign_key => {:on_delete => :cascade}
```
Comment (agrare) [inline db/migrate/20181108113645_add_tenant_id_to_regions_and_subscriptions.rb]: These should be `:null => false`

Comment (Ladas) [reply db/migrate/20181108113645_add_tenant_id_to_regions_and_subscriptions.rb]: right

File: app/models/source_region.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "archived_concern"
+
+class SourceRegion < ApplicationRecord
+  include ArchivedConcern
+
+  belongs_to :source
```
Comment (agrare) [inline app/models/source_region.rb]: These should belong to `:tenant` and `Tenant` should have many regions and subscriptions

**Comment by agrare** [comment]
Just the one thing then LGTM

File: app/models/source_region.rb (hunk 2 above)
Comment (Ladas) [reply app/models/source_region.rb]: done

**Pull request merged**